R/model_LSA.R

```diff
@@ -123,7 +124,8 @@ LatentSemanticAnalysis = R6::R6Class(
     n_topics = NULL,
     components_ = NULL,
     fitted = FALSE,
-    svd_method = NULL
+    svd_method = NULL,
+    vt = NULL
```

Comment:
can we also add `explained_ratio_` and `explained_variance_ratio_` fields as it is done is scikit-learn? It can be calculated similarly as you've shown, but a bit more efficiently:
```r
calculate_col_var = function(x) {
  colMeans(x * x) - colMeans(x) ^ 2
}
calculate_explained_var = function(x_original, x_low_rank) {
 col_var(x_low_rank) / sum(col_var(x_original))
}
```